tests/groupTree.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import sunbeam
+import datetimen
+
+spe3 = sunbeam.parse('spe3/SPE3CASE1.DATA')
+
+
+class TestGroupTree(unittest.TestCase):
+    spe3 = None
+
+    def setUp(self):
+        if self.spe3 is None:
+            self.spe3 = sunbeam.parse('spe3/SPE3CASE1.DATA')
+        time = es.schedule.start
+        self.groupTree = self.spe3.schedule.groupTree(time)
+
+    def test(self):
+        group = self.schedule.group('G1')
+        parent = self.groupTree.parent(group)
+        children = self.groupTree.children(group)
+
+        self.assertEqual('G0', parent)
+        self.assertEqual('G2', children[0])
```

Comment:
It does not look like this test has been hooked up here: https://github.com/Statoil/sunbeam/blob/master/tests/CMakeLists.txt - so I doubt it has run at all?